airflow/contrib/operators/dataproc_operator.py

```diff
@@ -660,15 +656,13 @@ def __init__(self,
                  dataproc_jars=None,
                  gcp_conn_id='google_cloud_default',
                  delegate_to=None,
-                 labels=None,
```

Comment:
Now that this is here I don't think we can remove it without breaking anyone who got it working on 1.10.4

Reply:
I will cherry-pick the actual commit that added labels

Reply:
https://github.com/apache/airflow/commit/4d58f36df3b533edfce873799dc161ee34571d63

Reply:
Oh, though labels never _worked_. right.

Reply:
Exactly